integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -107,37 +111,39 @@ public void mount(boolean blocking, boolean debug, String[] fuseOpts) {
         throw new FuseException("Unable to mount FS, return code = " + res);
       }
     } catch (Exception e) {
-      mounted.set(false);
+      mMounted.set(false);
       throw new FuseException("Unable to mount FS", e);
     }
   }
 
   private int execMount(String[] arg) {
-    return libFuse.fuse_main_real(this, arg.length, arg);
+    return mLibFuse.fuse_main_real(this, arg.length, arg);
   }
 
+  /**
+   * Umount the mount point of this Fuse Filesystem.
+   *
+   * @param force     whether to do a force umount
+   */
   public void umount(boolean force) {
-    if (!mounted.get()) {
+    if (!mMounted.get()) {
       return;
     }
-    LOG.info("Umounting {}", mountPoint);
+    LOG.info("Umounting {}", mMountPoint);
     try {
       umountInternal();
     } catch (FuseException e) {
-      LOG.error("Failed to umount {}", mountPoint, e);
+      LOG.error("Failed to umount {}", mMountPoint, e);
       throw e;
     }
-    mounted.set(false);
+    mMounted.set(false);
   }
 
   private void umountInternal() {
-    int exitCode = 1;
-    String mountPath = mountPoint.toString();
+    int exitCode;
+    String mountPath = mMountPoint.toString();
     if (SystemUtils.IS_OS_WINDOWS) {
-      // Pointer fusePointer = this.fusePointer;
-      // if (fusePointer != null) {
-      // libFuse.fuse_exit(fusePointer);
-      // }
```

Comment:
I have no idea what this chunk of commented code is doing plus why it's here. So just remove it and directly throw an exception, or let me know what we should do about it. The behavior is not changed.